s2/cellunion.go

```diff
@@ -553,7 +553,7 @@ func (cu *CellUnion) encode(e *encoder) {
 	}
 }
 
-// Decode decodes the CellUnion.
+// Decode decodes the CellUnion. The resulting CellUnion is not checked for validity.
```

Comment:
Apparently I can't comment on unchanged lines.

Also add a comment to Encode() that IsValid() is not a precondition.